### PR TITLE
Fixes `auth_message` so that it correctly renders HTML

### PR DESCRIPTION
**Pull request by abidlabs**
Quick fix and story test

Fixes: #7535 
Fixes: https://github.com/gradio-app/gradio/issues/7291

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
Looks good! Thanks @abidlabs!

**Comment by abidlabs** [comment]
Thanks @pngwn!

File: js/app/src/Login.svelte [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	<Column variant="panel" min_width={480}>
 		<h2>{$_("login.login")}</h2>
 		{#if auth_message}
-			<p class="auth">{auth_message}</p>
+			<p class="auth">{@html auth_message}</p>
```
Comment (pseudotensor) [inline js/app/src/Login.svelte:40]: I see, cool thanks.  I just tried to remove the whole class, I don't know this stuff enough.

Comment (abidlabs) [reply js/app/src/Login.svelte:40]: No problem thanks for flagging this!

**Pull request merged**